feg/gateway/services/n7_n40_proxy/servicers/session_controller.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2021 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package servicers
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+
+	"magma/feg/gateway/policydb"
+	n7_client "magma/feg/gateway/sbi/specs/TS29512NpcfSMPolicyControl"
+	"magma/feg/gateway/services/n7_n40_proxy/n7"
+	"magma/lte/cloud/go/protos"
+)
+
+type CentralSessionController struct {
+	policyClient *n7_client.ClientWithResponses
+	dbClient     policydb.PolicyDBClient
+	cfg          *SessionControllerConfig
+}
+
+type SessionControllerConfig struct {
+	N7Config       *n7.N7Config
+	RequestTimeout time.Duration
+}
+
+func NewCentralSessionController(
+	dbClient policydb.PolicyDBClient,
+) (*CentralSessionController, error) {
+	n7config := n7.GetN7Config()
+	if n7config == nil {
+		return nil, fmt.Errorf("invalid n7 config")
+	}
+	policyClient, err := n7.CreateN7Client(n7config.Server)
+	if err != nil {
+		glog.Errorf("Creating N7 Client failed: %s", err)
+		return nil, err
+	}
+	return &CentralSessionController{
+		policyClient: policyClient,
+		dbClient:     dbClient,
+		cfg: &SessionControllerConfig{
+			N7Config:       n7config,
+			RequestTimeout: 3 * time.Second,
```

Comment:
Never ever hardcode a value in middle of the code. It gets lost and it is impossible to find

Create a const like here 

https://sourcegraph.com/-/editor?remote_url=git%40github.com%3Amagma%2Fmagma.git&branch=master&file=feg%2Fgateway%2Fgtp%2Fgtp_client.go&editor=JetBrains&version=v1.2.1&utm_product_name=IntelliJ+IDEA&utm_product_version=2021.2.2&start_row=31&start_col=12&end_row=31&end_col=12